ERCS/erc-7806.md

```diff
@@ -0,0 +1,451 @@
+---
+eip: 7806
+title: Minimal intent-centric EOA smart account
+description: Minimal effort intent-centric interfaces for EOA account abstraction
```

Comment:
> Minimal effort

This doesn't really add a lot of information to the description. It's better to include specific/differentiating features than generally saying "minimal".

Reply:
Updated 👍